src/main/java/com/checkout/apm/klarna/KlarnaClientImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.checkout.apm.klarna;
+
+import com.checkout.AbstractClient;
+import com.checkout.ApiClient;
+import com.checkout.CheckoutConfiguration;
+import com.checkout.PublicKeyCredentials;
+import com.checkout.payments.VoidRequest;
+import com.checkout.payments.VoidResponse;
+
+import java.util.concurrent.CompletableFuture;
+
+import static com.checkout.common.CheckoutUtils.validateParams;
+
+public class KlarnaClientImpl extends AbstractClient implements KlarnaClient {
+
+    private static final String KLARNA = "klarna";
+    private static final String CREDIT_SESSIONS = "credit-sessions";
+    private static final String ORDERS = "orders";
+    private static final String CAPTURES = "captures";
+    private static final String VOIDS = "voids";
+
+    public KlarnaClientImpl(final ApiClient apiClient, final CheckoutConfiguration configuration) {
+        super(apiClient, PublicKeyCredentials.fromConfiguration(configuration));
+    }
+
+    @Override
+    public CompletableFuture<CreditSessionResponse> createSession(final CreditSessionRequest creditSessionRequest) {
```

Comment:
Should this operation be called `createCreditSession`? Just to be similar to `getCreditSession`.